server/controllers/FollowController.js

```diff
@@ -0,0 +1,201 @@
+   * @param {Object} req the request body
+   * @param {Object} res the response body
+   * @param {function} next a call to the next function
+   * @returns {Object} the response body
+   */
+  static follow(req, res, next) {
+    const { id: followerId } = req.body.decoded;
+    const followingId = parseInt(req.params.userId, 10);
+
+    // // Ensure a user cannot unfollow him/her self
+    if (followingId === followerId) {
+      return res.status(422).json({
+        errors: { message: 'You cannot follow yourself' },
+      });
+    }
+    // Get the user being followed
+    User.findById(followingId)
+      .then((user) => {
+        // Return error message if user is not found
+        if (!user) {
+          return res.status(404).json({
+            errors: { message: 'User not found' },
+          });
+        }
+        Follow.findOrCreate({ where: { followerId, followingId } })
+          .spread((userFollower, created) => {
+            // No action is performed if you already following an user
+            if (!created) {
+              return res.status(422).json({
+                errors: { message: 'You are already following this User' },
+              });
+            }
+            // Return user's profile
+            return res.status(201).json({
+              profile: userProfile(user, true),
+            });
+          });
+      }).catch(next);
+    return null;
+  }
+
+  /**
+   * remove a user from the follower list of an user
+   * @param {Object} req the request body
+   * @param {Object} res the response body
+   * @param {function} next a call to the next function
+   * @returns {Object} the response body
+   */
+  static unfollow(req, res, next) {
+    const { userId: followingId } = req.params;
+    const { id: followerId } = req.body.decoded;
+
+    // Get the user being followed
+    User.findById(followingId)
+      .then((user) => {
+        // Return error message if user is not found
+        if (!user) {
+          return res.status(404).json({
+            errors: { message: 'User not found' },
+          });
+        }
+        // Remove user-follower relationship
+        user.removeFollower(followerId)
+          .then((follow) => {
+            if (!follow) {
+              return res.status(404).json({
+                errors: {
+                  message: 'You are not following this author',
+                },
+              });
+            }
+            // Return user's profile
+            return res.status(200).json({
+              profile: userProfile(user, false),
+            });
+          });
+      }).catch(next);
+    return null;
+  }
+
+  /**
+   * returns all the followers of a login user
+   * or the specified user
+   * @param {Object} req the request body
+   * @param {Object} res the response body
+   * @param {function} next a call to the next function
+   * @returns {Object} the response body
+   */
+  static followers(req, res, next) {
```

Comment:
You should give your controller functions more intuitive names, for this case, you can name it `getFollowers`